wordfugue/blog/urls.py

```diff
@@ -1,6 +1,8 @@
 from django.conf.urls import url
 from . import views
 
+app_name = 'blog'
```

Comment:
To use the namespaces in Django URLs for Django 2.1, I need to define an `app_name` in the url files for the django apps. Namespaces are a way of telling django, "Hey, I want to reference this URL that belongs to this app, out of the full set of URLs you know about"